src/main/resources/templates/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Enter Name & Password</title>
+</head>
+<body>
+    <form action="/create-user" method="post">
```

Comment:
If the form is all that is in here it might be better to just have a `login.html`